yarn-project/circuits.js/src/abis/abis.ts

```diff
@@ -15,7 +14,7 @@ import {
   Vector,
 } from '../index.js';
 import { serializeBufferArrayToVector } from '../utils/serialize.js';
-import { CircuitsWasm } from '../wasm/index.js';
+import { IWasmModule } from '@aztec/foundation/wasm';
```

Comment:
A bunch of the existing WASM structures have been collapsed into just CircuitsWasm has a WasmModule, abstracted by an IWasmModule (which will be shared by upstream bb.js soon)